### ISSUE TITLE
While in a call, if you hide the chat, you cannot make it appear again (NC 26)

### ISSUE THREAD

**Issue by maximelehericy** [nextcloud/spreed#8884]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1.
2.
3.

### Expected behaviour
I hide the chat while being in a video.
I should have a button allowing me to reopen it.

### Actual behaviour
I hide the chat right bar.
I have no option to reopen it.

## Talk app
c.nc.c 26 beta
**Talk app version:** (see apps admin page: `/index.php/settings/apps`)

**Custom Signaling server configured:** yes/no and version (see additional admin settings: `/index.php/index.php/settings/admin/talk#signaling_server`)

**Custom TURN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/talk#turn_server`)

**Custom STUN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/talk#stun_server`)

## Browser
Firefox
**Microphone available:** yes/no

**Camera available:** yes/no

**Operating system:** Windows/Ubuntu/...

**Browser name:** Firefox/Chrome/...

**Browser version:** 85/96/...

### Browser log

<details>
```
Insert your browser log here, this could for example include:
a) The javascript console log
b) The network log
c) ...
```

</details>

## Server configuration

**Operating system**: Ubuntu/RedHat/...

**Web server:** Apache/Nginx

**Database:** MySQL/Maria/SQLite/PostgreSQL

**PHP version:** 7.4/8.0/8.1

**Nextcloud Version:** (see admin page)

**List of activated apps:**

<details>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your server installation folder
```
</details>

**Nextcloud configuration:**

<details>

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder
```
</details>

### Server log (data/nextcloud.log)
<details>

```
Insert your server log here
```
</details>

**Comment by nickvergessen** [comment]
![image](https://user-images.githubusercontent.com/213943/221880533-abdd72cf-e265-4e40-ba43-50e467034083.png)

Works fine here.

Can you show a screenshot of your top right corner?

**Comment by maximelehericy** [comment]
yeah @nickvergessen , I noticed it was back in an other call afterwards, but now I found how to reproduce :)
![image](https://user-images.githubusercontent.com/87175513/222172662-46fdd740-6cff-421c-99c9-e3d48bb8f05a.png)

start/join a call.
open the chat.
close the chat
open the chat,
close the chat.

Until now, everything is fine.

Now, click the three dots, then breakout rooms.
close the breakout room popup
close the chat
the icon to reopen the chat is not displayed.

I then need to ctrl+F5 and rejoin the call to make the button reappear.
